core/indexer/fulltext.go

```diff
@@ -31,9 +32,12 @@ func (i *indexer) ForceFTIndex() {
 	}
 }
 
-func (i *indexer) ftLoop() {
+// ftLoop runs full-text indexer
+// MUST NOT be called more than once
```

Comment:
it's better to use context with cancel, it could be closed multiple times

Reply:
this method existing only to extract the code for the goroutine and can must called only once. Also there is a comment